src/mapper/cmd/main.go

```diff
@@ -76,7 +80,7 @@ func main() {
 	mapperServer.Use(middleware.Logger())
 
 	// start manager with operators
-	mgr, err := manager.New(clientconfig.GetConfigOrDie(), manager.Options{MetricsBindAddress: "0"})
+	mgr, err := manager.New(clientconfig.GetConfigOrDie(), manager.Options{})
```

Comment:
The "0" disables the metrics bind address. If you remove it, it will bind to the default, which is not what we want (we run a separate metrics server)

Reply:
fixed (the api changed)